libs/ui-lib/lib/ocm/components/AddHosts/day2Wizard/Day2ClusterDetails.tsx

```diff
@@ -104,6 +107,41 @@ const Day2ClusterDetails = () => {
     [day2Cluster.id, wizardContext],
   );
 
+  const onChangeCpuArchitectureDropdown = React.useCallback(
+    (value: string, initialValues: Day2ClusterDetailValues): void => {
```

Comment:
Can we make this async?

Reply:
If I put this method async,then I reveiced an error when I use in onChange (because we need a void method):

<CpuArchitectureDropdown
                    openshiftVersion={day2Cluster.openshiftVersion}
                    day1CpuArchitecture={initialValues.cpuArchitecture}
                    cpuArchitectures={cpuArchitectures}
                    onChange={(value) => onChangeCpuArchitectureDropdown(value, initialValues)}
                  />